docs/src/main/sphinx/admin/graceful-shutdown.rst

```diff
@@ -23,9 +23,12 @@ Keep the following aspects in mind:
 * If you have TLS/HTTPS enabled, you have to ensure the worker certificate is
   CA signed, or trusted by the server calling the shut down endpoint.
   Otherwise, you can make the call ``--insecure``, but that isn't recommended.
-* If :ref:`system information rules <system-file-auth-system_information>` are
-  configured, then the user in the HTTP request must have read and write
-  permissions in the system information rules.
+* The ``default`` :doc:`/security/built-in-system-access-control` does not allow
+  graceful shutdowns. You can use  the ``allow-all`` system access control, or
```

Comment:
```suggestion
  graceful shutdowns. You can use the ``allow-all`` system access control, or
```
The wording on this is a little weird. Maybe "You can set the `access-control.name` configuration property to `allow-all`, or you can..."

Reply:
I don't want to document configure the access control .. thats in the linked page.